client/download/download.tsx

```diff
@@ -81,46 +83,54 @@ const InstallerButtonLabel = styled.span`
   align-items: center;
 `
 
-export default class Download extends React.Component {
-  override render() {
-    return (
-      <React.Fragment>
-        <Blurb>Download the ShieldBattery client to:</Blurb>
-        <BlurbList>
-          <li>Play games</li>
-          <li>Watch replays</li>
-          <li>Explore maps</li>
-          <li>And more!</li>
-        </BlurbList>
-        <InstallerLinks>
-          <RaisedButton
-            onClick={this.onDownloadClick}
-            iconStart={<MaterialIcon icon='download' />}
-            label={<InstallerButtonLabel>Download client</InstallerButtonLabel>}
-          />
-        </InstallerLinks>
-        <SupportText>Want to support the project?</SupportText>
-        <SupportLinks>
-          <a
-            href='https://github.com/sponsors/ShieldBattery'
-            target='_blank'
-            rel='noopener'
-            title='GitHub Sponsors'>
-            <StyledGithubIcon />
-          </a>
-          <a href='https://ko-fi.com/tec27' target='_blank' rel='noopener' title='Ko-fi'>
-            <StyledKofiIcon />
-          </a>
-          <a href='https://patreon.com/tec27' target='_blank' rel='noopener' title='Patreon'>
-            <StyledPatreonIcon />
-          </a>
-        </SupportLinks>
-      </React.Fragment>
-    )
-  }
+export function Download() {
+  const { t } = useTranslation()
 
-  onDownloadClick = () => {
+  const onDownloadClick = useStableCallback(() => {
```

Comment:
This is a memory/runtime cost for 0 gain, just use a normal function

The rule is: If you are creating a component that may be used in a bunch of places and you can't be sure when re-renders will matter, memoize. If you are creating a top-level-ish component and you need to be sure components that you're passing functions to don't re-render if their other props don't change, memoize.

This case is neither, this is a top-level component that is pretty shallow and doesn't care about re-rendering (and, in fact, has no props that can even change so...)